.eslintrc.js

```diff
@@ -165,6 +165,7 @@ module.exports = {
                 ],
                 curly: 'error',
                 'you-dont-need-lodash-underscore/throttle': 'off',
+                'react/prop-types': 'off',
```

Comment:
Explanation in the other [comment](https://github.com/Expensify/App/pull/29763/files#r1362044572)